### PR TITLE
Added GET/questions route.

### PR DESCRIPTION
**Pull request by farhanali3193**
@jabrake @tchitrakorn:
1. Successfully getting all questions from only one table.
2. Need to implement a query which gets answers and photos as well. Will do it later.

### CODE REVIEW

**Comment by farhanali3193** [comment]
1. Successfully getting all answers only from the answers table.

File: server/index.js [hunk 1]
```diff
@@ -5,11 +5,59 @@ const path = require('path');
...
+    })
+    .catch((err) => {
+      console.log('ERROR IN SERVER GETTING ALL QUESTIONS', err)
+    });
+});
+
+//------
+//GET ALL ANSWERS
+
+app.get(`/qa/questions/:question_id/answers`, (req, res) => {
```
Comment (jabrake) [inline server/index.js:40]: I'm assuming this works, but curious how the :question_id fragment in the string will function once that's replaced by a number in a real scenario?

File: pg/index.js [hunk 2]
```diff
@@ -4,14 +4,32 @@ const client = new Client({
...
-client.query(text)
-  .then((res) => {
-    console.log('RESPONSE', res.rows);
-  })
-  .catch((err) => {
-    console.log('ERROR GETTING RESPONSE', err);
-  })
\ No newline at end of file
+let getQuestions = (product_id, offset, limit) => {
+  let questionsQuery = `SELECT * FROM questions WHERE product_id = ${product_id} ORDER BY question_id ASC OFFSET ${offset} LIMIT ${limit}`;
```
Comment (tchitrakorn) [inline pg/index.js:11]: Based on the documentation (https://node-postgres.com/features/queries), my understanding is that this method (i.e., direct string concatenation in query text) may lead to SQL injection vulnerabilities? I created a separate variable to store the values and then pass it in as a second argument in the first then clause. Not sure if this approach has any obvious advantages though!

File: pg/index.js [hunk 3]
```diff
@@ -4,14 +4,32 @@ const client = new Client({
...
   .catch((err) => console.log('ERROR CONNECTING TO PG', err))
 
-let text = `SELECT * FROM questions offset 3518960`;
-client.query(text)
-  .then((res) => {
-    console.log('RESPONSE', res.rows);
-  })
-  .catch((err) => {
-    console.log('ERROR GETTING RESPONSE', err);
-  })
```
Comment (tchitrakorn) [inline pg/index.js:17]: Maybe this is a stylistic choice, but there's also a promise option for this (My apologies if you're already aware of this!)

File: server/index.js (hunk 1 above)
Comment (farhanali3193) [reply server/index.js:40]: @jabrake That's a good question. But I thought the Client will always send the request in a string format. The client might do something like this:
`axios.get('API_URL/qa/questions/1234/answers')`
Where 1234 should be a string now.

File: server/index.js [hunk 4]
```diff
@@ -5,11 +5,59 @@ const path = require('path');
 const bodyParser = require('body-parser');
 const cors = require('cors');
 const AWS = require("aws-sdk");
```
Comment (tchitrakorn) [inline server/index.js:7]: If you have some time, could you please clarify what you need these two for? I know that cors is a crucial concept but I have never used the module before. Also, where would AWS come in? Thank you so much in advance for the explanation!

**Review by tchitrakorn — APPROVED** [review]

File: pg/index.js (hunk 2 above)
Comment (farhanali3193) [reply pg/index.js:11]: Got it. I will look into it

**Pull request merged**